src/mnemonic.rs

```diff
@@ -221,9 +226,12 @@ impl Mnemonic {
 
     /// Consume the `Mnemonic` and return the phrase as a `String`.
     ///
-    /// This operation doesn't perform any allocations.
-    pub fn into_phrase(self) -> String {
-        self.phrase
+    /// This operation creates an empty string and swaps values with the mnemonic's phrase. This
```

Comment:
I'd be tempted to leave this out of the doc comments, since it's quite internal. Might be better as a `//` comment inside the function. Up to you though :)

Reply:
I think that's a good point. Updated. 👍